### ISSUE TITLE
Call to undefined method Doctrine\ODM\PHPCR\DocumentManager::transactional()

### ISSUE THREAD

**Issue by delportd** [symfony-cmf/symfony-cmf#217]
Hi !

I'm using this good CMF for the second time, but when I try to load the fixtures with "php app/console doctrine:phpcr:fixtures:load" to initialise the project (as they say in the documentation http://symfony.com/doc/master/cmf/book/installation.html) it fails.

The error : 
```bash
root@56d55021afb3:/var/www# php app/console doctrine:phpcr:fixtures:load
Careful, database will be purged. Do you want to continue Y/N ?y
PHP Fatal error:  Call to undefined method Doctrine\ODM\PHPCR\DocumentManager::transactional() in /var/www/vendor/doctrine/data-fixtures/lib/Doctrine/Common/DataFixtures/Executor/PHPCRExecutor.php on line 63
                                                                       
  [Symfony\Component\Debug\Exception\UndefinedMethodException]                             
  Attempted to call method "transactional" on class "Doctrine\ODM\PHPCR\DocumentManager".  

doctrine:phpcr:fixtures:load [--fixtures[="..."]] [--append] [--name[="..."]] [--no-initialize] [--session[="..."]] [--dm[="..."]]
```

My parameters.yml
```
parameters:
    database_driver: pdo_mysql
    database_host: 172.17.0.50
    database_port: 3306
    database_name: symfony2
    database_user: root
    database_password: PYNhXClUoAZMWxgAY5mfScd5Kvcf3m
    database_path: null
    phpcr_backend:
        type: doctrinedbal
        connection: default
        caches:
            meta: doctrine_cache.providers.phpcr_meta
            nodes: doctrine_cache.providers.phpcr_nodes
    phpcr_workspace: default
    phpcr_user: admin
    phpcr_password: admin
    mailer_transport: smtp
    mailer_host: 127.0.0.1
    mailer_user: null
    mailer_password: null
    locale: en
    secret: Bv8sL3G1OsNazsKyftEmFXwds8RcBr
    debug_toolbar: true
    debug_redirects: false
    use_assetic_controller: true
```

The thing was working yesterday (I'm seting up an dev environment in docker) I didn't changed anything to my Dockerfiles, so I think the problem may be coming from some vendor library (maybe doctrine/data-fixtures) ?

The CMF was working with : 
```
"name": "doctrine/data-fixtures",
            "version": "v1.0.1",
```
But not anymore with : 
```
"name": "doctrine/data-fixtures",
            "version": "v1.1.0",
```

Thank you,
A french guy.

**Comment by delportd** [comment]
Downgrading to doctrine/data-fixtures v1.0.1 seems to solve the problem, does the maintainer of the repository needs to know ?

```bash
root@6ccc98d5825f:/var/www# composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
  - Installing doctrine/data-fixtures (v1.0.1)
    Downloading: 100%         

doctrine/data-fixtures suggests installing doctrine/mongodb-odm (For loading MongoDB ODM fixtures)
doctrine/data-fixtures suggests installing doctrine/orm (For loading ORM fixtures)
Generating autoload files
Updating the "app/config/parameters.yml" file
Download or update create
Download or update ckeditor
Clearing the cache for the dev environment with debug true
Installing assets as hard copies.
Installing assets for Symfony\Bundle\FrameworkBundle into web/bundles/framework
Installing assets for Sonata\CoreBundle into web/bundles/sonatacore
Installing assets for Symfony\Cmf\Bundle\CreateBundle into web/bundles/cmfcreate
Installing assets for Acme\DemoBundle into web/bundles/acmedemo
Installing assets for Sensio\Bundle\DistributionBundle into web/bundles/sensiodistribution
root@6ccc98d5825f:/var/www# 
root@6ccc98d5825f:/var/www# php app/console doctrine:phpcr:fixtures:load
Careful, database will be purged. Do you want to continue Y/N ?y
  > purging database
  > Executing initializer: CmfContentBundle
  > Executing initializer: CmfSimpleCmsBundle Homepage
  > Executing initializer: CmfBlockBundle
  > Executing initializer: CmfMenuBundle
  > loading Acme\DemoBundle\DataFixtures\PHPCR\LoadDemoData
root@6ccc98d5825f:/var/www#
```

**Comment by lsmith77** [comment]
This seems to be a regression in the fixtures library. try using ``1.0.*`` while we investigate this.

/cc @Ocramius @lavoiesl

**Comment by lsmith77** [comment]
I guess once we release https://github.com/doctrine/phpcr-odm/pull/582 this will work but it would still make sense to handle this regression.

**Comment by dbu** [comment]
yeah, either fixtures must declare a conflicts in their composer.json or
do a method_exists check before calling that.

**Comment by dbu** [comment]
proposed a fix: https://github.com/doctrine/data-fixtures/pull/190

**Comment by dbu** [comment]
fixed and released as data fixtures 1.1.1

**Comment by delportd** [comment]
Thank you very much !
